data/mapcss-mapping.csv

```diff
@@ -1133,7 +1133,7 @@ man_made|water_tower;1051;
 highway|world_level;1052;
 highway|world_towns_level;1053;
 building|train_station;1054;
-route|shuttle_train;1055;
+route|shuttle_train;[route=shuttle_train],[route=train][shuttle];;;;1055;
```

Comment:
maybe it'd be better to remove all useless `;name;int_name;` in one go, as single changes like this can confuse people?